python/ray/autoscaler/commands.py

```diff
@@ -422,6 +429,28 @@ def _exec(updater, cmd, screen, tmux, expect_error=False, port_forward=None):
             port_forward=port_forward)
 
 
+def copy_to_docker(config_file, target_script, override_cluster_name):
+    """Rsyncs files.
+
+    Arguments:
+        config_file: path to the cluster yaml
+        target: target file on host
+        override_cluster_name: set the name of the cluster
+    """
+    config = yaml.load(open(config_file).read())
+    if override_cluster_name is not None:
+        config["cluster_name"] = override_cluster_name
+    config = _bootstrap_config(config)
+    base_path = os.path.basename(target_script)
+    cname = config["docker"]["container_name"]
+    cmd = "docker cp {} {}:{}".format(target_script, cname, base_path)
```

Comment:
You might want to wrap the target with quotes, so that paths with `~` work too:
```
    cmd = "docker cp {} {}:\"{}\"".format(target_script, cname, base_path)
```